### PR TITLE
Fix depwarn on 0.6 due to vectorized functions

### PR DESCRIPTION
**Pull request by yuyichao**
Requires JuliaLang/Compat.jl#280 and a new `Compat` tag.

### CODE REVIEW

**Comment by yuyichao** [comment]
Compat.jl PR merged and tagged. Version requirements updated, run travis on 0.4, 0.5, nightly and CI passed. Should be ready to go.

**Comment by timholy** [comment]
Thanks, @yuyichao!

File: appveyor.yml [hunk 1]
```diff
@@ -1,7 +1,9 @@
 environment:
   matrix:
-  - JULIAVERSION: "julialang/bin/winnt/x86/0.3/julia-0.3-latest-win32.exe"
-  - JULIAVERSION: "julialang/bin/winnt/x64/0.3/julia-0.3-latest-win64.exe"
+  - JULIAVERSION: "julialang/bin/winnt/x86/0.4/julia-0.3-latest-win32.exe"
+  - JULIAVERSION: "julialang/bin/winnt/x64/0.4/julia-0.3-latest-win64.exe"
+  - JULIAVERSION: "julialang/bin/winnt/x86/0.5/julia-0.3-latest-win32.exe"
+  - JULIAVERSION: "julialang/bin/winnt/x64/0.5/julia-0.3-latest-win64.exe"
```
Comment (tkelman) [inline appveyor.yml]: these links are wrong. appveyor isn't running here, needs to be enabled

Comment (timholy) [inline appveyor.yml]: #55

**Pull request merged**